protocol/0038-OLIQ-liquidity_provision_order_type.md

```diff
@@ -60,10 +60,20 @@ Input data:
 
 Steps:
 
+1. From the market parameter `market.liquidity.volumeRange` which is a percentage price move (e.g. `0.05 = 5%` and from `mid_price` calculate:
```

Comment:
Just to confirm, is that a parameter of the market or a network parameter?

Reply:
I've updated the statement - this is now 100% clear that it's market parameter.